edx/config/lms/docker_run.py

```diff
@@ -65,7 +65,7 @@
 ################################ ALWAYS THE SAME ##############################
 
 RELEASE = config("RELEASE", default=None)
-DEBUG = False
+DEBUG = True
```

Comment:
Why do you switch to Debug mode?

Reply:
It seems that some exception messages tend to be more verbose with debug mode.
Also, some security checks are bypassed, for example [here](https://github.com/openfun/edx-platform/blob/dogwood.3-fun/openedx/core/lib/api/permissions.py#L28).